spec/services/analytics_spec.rb

```diff
@@ -83,6 +85,19 @@
       analytics.track_event('Trackable Event', user_id: tracked_user.uuid)
     end
 
+    context 'irs_session_id' do
```

Comment:
we might want to have a negative test case, when `irs_session_id` is nil, that the payload doesn't even contain the key

Reply:
I had initially started to write this and then realized that it would have broken the existing tests if that weren't true.

But, let me write one explicitly for completeness. I think part of the value in tests is in what they assert is expected behavior and we want to call this out as such.